exec/totemconfig.c

```diff
@@ -1912,6 +1912,8 @@ extern int totem_config_read (
 			       "255.255.255.255", TOTEM_IP_VERSION_4);
 	}
 
+	totem_config->ip_dscp = 0;
+	icmap_get_uint8("totem.ip_dscp", &totem_config->ip_dscp);
```

Comment:
coverity failed on this line - error code should be checked, but for this case I would recommend to just prepend (void) to let coverity (and reader of the code) know that we don't care about return code.

Reply:
@dahast: Coverity is the difference. Sometimes it finds something somewhere and not on other places. The model it uses is simplified. For this specific case it probably notifies `icmap_get_uint8` return value was checked somewhere else so it expects it should always be checked. Implement proper checking for that specific code is complete non-sense so ignoring return value is best option.